app/resources/forms.py

```diff
@@ -0,0 +1,39 @@
+from flask.ext.wtf import Form
+from wtforms.fields import (
+    StringField,
+    IntegerField,
+    SubmitField
+)
+from wtforms.validators import InputRequired, Length
+
+
+class ResourceForm(Form):
+    autocomplete = StringField('Enter the address')
```

Comment:
Is this for address autocomplete? If so, could it be called `address_autocomplete`?